cmd/kwil-cli/cmds/database/read_schema.go

```diff
@@ -49,8 +49,6 @@ func readSchemaCmd() *cobra.Command {
 		},
 	}
 
-	cmd.Flags().StringP(nameFlag, "n", "", "the target database name")
-	cmd.Flags().StringP(ownerFlag, "o", "", "the target database owner")
-	cmd.Flags().StringP(dbidFlag, "i", "", "the target database id")
+	bindFlagsTargetingDatabase(cmd)
```

Comment:
the description of this command is indented, better keep it the same as other cmds.

Reply:
good catch